README.md

```diff
@@ -1,15 +1,59 @@
-# Flat_scrape_tests
-Extract data about existing flat offers
+# Scrapy scripts for flat data extraction
 
-Some notes:
-1. I just need to use Python to extract data about flats price/size/location/... into a nice table. Data merging/cleaning/analysis/plots will be done later with R (not sure why I ended up with this option precisely)
+Scripts for extracting information about existing flat offers in Kaunas from two websites:
 
-2. Flat selling and rent crawlers are identical more or less. So main files that would be of some interest are        -aruodas/aroudas/spiders/aruodascrawl.py
--capitallt/capitallt/spiders/capitallt_scraper.py
+https://www.aruodas.lt/
 
-3. To run scripts I used command in terminal, e.g.:
-scrapy crawl capitallt -O capitalltbutai.csv
+and
 
-4. Current examples of extracted tables are 
--capitallt/capitalltbutai.csv
+https://www.capital.lt/lt/nekilnojamas-turtas/
+
+
+## Description
+
+I want to have an idea what is flat situation in Kaunas. Specifically, I am interested in flat's:
+1. Location (Rajonas)
+2. Street (Gatve)
+3. Price (Kaina)
+4. Size (Dydis)
+5. Number of rooms (Kambariai)
+6. Year (Metai)
+7. Floor/How many floors (Aukstas/is)
+8*. (Just in case) I will be saving URL of each add (Nuoroda) 
+
+This information should be put in a csv table, that can be later cleaned, merged, analyzed and plotted (I will be using R for this and these scripts are not added here so far)
+
+## Examples of extracted tables
+
+![Alt text](/Screenshot_of_table.png?raw=true "Screenshot of extracted table")
```

Comment:
Excellent! 
Ideally, remove the orthographic checker to avoid having all words being underlined

Reply:
file should be put in an `img/` folder, for example